src/test/kotlin/com/zopa/ktor/opentracing/KtorOpenTracingTest.kt

```diff
@@ -51,12 +51,13 @@ class KtorOpenTracingTest {
 
     @Test
     fun `Create server span for request without trace context in headers`() = withTestApplication {
+        val configuredPath = "/greeting/{id}"
```

Comment:
`routePath` would be more precise? Would you consider having another param in this path ( that is not an UUID ) to document the behaviour change

Reply:
Yes good idea, will add